openpathsampling/engines/openmm/engine.py

```diff
@@ -11,6 +11,35 @@
 
 logger = logging.getLogger(__name__)
 
+def restore_custom_integrator_interface(integrator):
+    """
+    Restore the interface to a custom integrator. 
+
+    When subclasses of CustomIntegrator are deserialized, they only show up
+    as `CustomIntegrator`, and therefore lose any additional interface
+    provided by the subclass.
+
+    So far, only subclasses of `openmmtools.ThermostatedIntegrator` are
+    supported. Hopefully, we can establish some sort of more widely-used
+    approach based on the tricks established in there.
+    """
+    # for openmmtools integrators
+    try:
+        import openmmtools
```

Comment:
is it possible that you need to `import openmmtools.integrators` here? travis cannot find `ThermostatedIntegrator`